packages/website/src/components/search/search.css

```diff
@@ -1,29 +1,81 @@
-.ma-search {
-  column-gap: var(--basis-space-inline-md);
-  container-name: ma-search;
-  container-type: inline-size;
+/* stylelint-disable -- Overwriting DocSearch, we don't have control over the markup. */
+.DocSearch {
+  --ma-docsearch-button-border-color: var(--utrecht-button-secondary-action-border-color);
+  --ma-docsearch-button-border-radius: 8px;
+  --ma-docsearch-button-font-family: var(--utrecht-button-font-family);
+  --ma-docsearch-button-hover-background-color: var(--utrecht-button-secondary-action-hover-background-color);
+  --ma-docsearch-button-hover-border-color: var(--utrecht-button-secondary-action-hover-border-color);
+  --ma-docsearch-button-hover-color: var(--utrecht-button-secondary-action-hover-color);
+  --ma-docsearch-button-key-background-color: var(--basis-color-accent-1-bg-default);
+  --ma-docsearch-button-key-color: inherit;
+  --ma-docsearch-button-key-hover-background-color: var(--basis-color-action-1-inverse-bg-default);
+  --ma-docsearch-button-key-hover-color: white;
+  --ma-docsearch-button-key-active-background-color: var(--basis-color-action-1-inverse-bg-default);
+  --ma-docsearch-button-key-active-color: white;
+  --ma-docsearch-button-key-focus-visible-background-color: black;
+  --ma-docsearch-button-key-focus-visible-color: white;
+  --ma-docsearch-button-padding-block-end: 12px;
+  --ma-docsearch-button-padding-block-start: 12px;
+  --ma-docsearch-button-padding-inline-end: 16px;
+  --ma-docsearch-button-padding-inline-start: 16px;
```

Comment:
Kan dit niet onderdeel worden van de theme, in een andere file? Huisstijl en component scheiden?